src/base/torrentfilter.h

```diff
@@ -38,7 +38,7 @@ namespace BitTorrent
     class Torrent;
 }
 
-using InfoHashSet = QSet<BitTorrent::InfoHash>;
+using SHA1HashSet = QSet<BitTorrent::InfoHash>;
```

Comment:
Seems we should use `InfoHashSet` as before. `TorrentFilter` doesn't need to know about sha* hashes.

Reply:
Fixed.
These are all remnants of my original changes. Fixing them requires more effort than the payload...